mirrord-layer/src/pod_api.rs

```diff
@@ -412,8 +381,252 @@ async fn create_job_pod_agent(
         .items
         .first()
         .and_then(|pod| pod.metadata.name.clone())
-        .ok_or(LayerError::PodNotFound(mirrord_agent_job_name))?;
+        .ok_or(LayerError::JobPodNotFound(mirrord_agent_job_name))?;
 
     wait_for_agent_startup(pods_api, &pod_name, "mirrord-agent".to_string()).await?;
     Ok(pod_name)
 }
+
+#[derive(Debug, Clone)]
+pub(crate) struct Response {
```

Comment:
It seems weird to have a struct called `Response` here since it's not a communication library. Is there a name that makes more sense?

Reply:
`TargetResult`?